### ISSUE TITLE
Shadbala values not matching

### ISSUE THREAD

**Issue by AbhayVivek** [VicharaVandana/jyotishyamitra#2]
I have matched the shadbala values with astrosage.com

> ### Few values are not matching

The JSON file of the values generated by jyotishyamitra is attached below : 
[balas.json](https://github.com/VicharaVandana/jyotishyamitra/files/14534227/balas.json)

The output generated by astrosage is as follows 
![image](https://github.com/VicharaVandana/jyotishyamitra/assets/143114904/a0bae865-de31-470c-beed-d4955190f003)

**Comment by VicharaVandana** [comment]
Mine is as per bphs. I cannot comment on astrosage software

**Comment by AbhayVivek** [comment]
I have matched the values with **Horosoft astrology software** also and they are **not coming same**

Did you also match the values with some software ?

**Comment by VicharaVandana** [comment]
No. I have not. But with my shadbala i am getting good predictions for my
practice.

**Comment by AbhayVivek** [comment]
Can you please help me out , 
i am looking for a code that generates accurate values 

If possible can you have a look at the code , 
i have little coding knowledge , that's why i cannot understand where is the problem

**Comment by VicharaVandana** [comment]
Sorry I can't.
